Apps/W1/Shopify/app/src/Base/Codeunits/ShpfyUpgradeMgt.Codeunit.al

```diff
@@ -347,6 +348,46 @@ codeunit 30106 "Shpfy Upgrade Mgt."
         exit(CreateDateTime(DMY2Date(1, 8, 2022), 0T));
     end;
 
+
+    procedure AddRetentionPolicyAllowedTables()
+    begin
+        AddRetentionPolicyAllowedTables(false);
+    end;
+
+    procedure AddRetentionPolicyAllowedTables(ForceUpdate: Boolean)
+    var
+        Field: Record Field;
+        RetenPolAllowedTables: Codeunit "Reten. Pol. Allowed Tables";
+        UpgradeTag: Codeunit "Upgrade Tag";
+        IsInitialSetup: Boolean;
+    begin
+        IsInitialSetup := not UpgradeTag.HasUpgradeTag(GetShpfyTablesAddedToAllowedListUpgradeTag());
+        if not (IsInitialSetup or ForceUpdate) then
+            exit;
+
+        RetenPolAllowedTables.AddAllowedTable(Database::"Shpfy Log Entry", Field.FieldNo(SystemCreatedAt), 7);
```

Comment:
@JesperSchulz could you maybe find someone to decide which field we should use? Field `"Date and Time"` or `SystemCreatedAt`?

And maybe ad someone to review.